.github/workflows/gateway-sharness.yml

```
@@ -43,6 +43,9 @@ jobs:
       - name: Install sharness dependencies
         run: make test_sharness_deps
         working-directory: kubo
-      - name: Run Sharness Tests
+      - name: Run Kubo Sharness Tests
         run: find . -maxdepth 1 -name "*gateway*.sh" -print0 | xargs -0 -I {} bash -c "echo {}; {}"
         working-directory: kubo/test/sharness
+      - name: Run Kubo CLI Tests
+        run: go test -v *gateway*.go
```

```suggestion
        run: go test -v -run=Gateway .
```
Pointing to individual go files don't compile the full package. It happen to work right now, but it would be confusing if this test started breaking because of a change in Kubo where some internal function of the package is moved (or other).